examples/fingerprint.py

```diff
@@ -28,7 +28,7 @@ def rf_model_builder(model_dir, hyperparams, mode):
 
 
 def load_model(args, tasks, hyperparams):
-  if args['dataset'] in ['BACE_classification']:
+  if args['dataset'] in ['BACE_classification', 'BBBP']:
```

Comment:
One thought here is do we want to build a general benchmarking script? `examples/fingerprint.py` and `examples/gnn.py` share a lot of code. It might be useful to build one general purpose script that we can use for multiple models.

Reply:
For forward compatibility, we can definitely use `examples/fingerprint.py` for other non-graph models and `examples/gnn.py` for other graph-based models. Nevertheless, it might be fine to keep these two files for the time being.